lib/perl/Genome/Model/Tools/Varscan/SomaticFilterWorkflow.t

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env genome-perl
+
+use strict;
+use warnings;
+
+BEGIN {
+    $ENV{UR_DBI_NO_COMMIT} = 1;
+    $ENV{UR_USE_DUMMY_AUTOGENERATED_IDS} = 1;
+    $ENV{NO_LSF} = 1;
+};
+
+use above "Genome";
+use Test::More;
+
+use_ok('Genome::Model::Tools::Varscan::SomaticFilterWorkflow') or die;
+my $expected_output_dir = $ENV{"GENOME_TEST_INPUTS"} .
```

Comment:
As I understand compare_ok under Genome::Utility::Test compares two files, is there a recommended way of comparing corresponding files in two directories ?